apps/authentication/models.py

```diff
@@ -313,7 +313,10 @@ def image(self):
 
     def get_visible_as_attending_events(self):
         """ Returns the default value of visible_as_attending_events set in privacy/personvern """
-        if hasattr(self, "privacy"):
```

Comment:
It is okay to remove this `hasattr`-check, since we create privacy for everyone with the migration